src/plugins/plugin-cesium/plugin-cesium.js

```diff
@@ -0,0 +1,84 @@
+import buildCartoLayerCesium from './carto-layer-cesium';
```

Comment:
buildCartoLayerCesium is like iLikeLongNames. Maybe you can call it cartoLayer, like the Leaflet plugin